arbitrum-docs/how-arbitrum-works/04-state-transition-function/01-stf-gentle-intro.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: 'A gentle introduction'
+description: 'Learn the fundamentals of Nitro, Arbitrum stack.'
+author: petevielhaber
+sme: Mehdi
+user_story: As a current or prospective Arbitrum user, I need learn more about Nitros design.
+content_type: get-started
+---
+
+A <a data-quicklook-from="state-transition-function">State Transition Function</a> (STF) is a core mechanism in <a data-quicklook-from="blockchain">blockchain</a> systems that defines how the system evolves from one state to another when transactions are processed. At its essence, the STF takes the current state of the blockchain–a comprehensive snapshot of account balances, <a data-quicklook-from="smart-contract">smart contract</a> data, and other ledger information–and an input (such as a <a data-quicklook-from="transaction">transaction</a> or a block of transactions) and deterministically computes the new state. This deterministic property is crucial because it ensures that all nodes in a decentralized network arrive at the same result when applying transactions, thereby maintaining consensus across the system.
+
+With <a data-quicklook-from="arbitrum">Arbitrum</a>, the STF plays an even more pivotal role. Offchain transactions are executed in batches according to this function, with the STF periodically submitting a concise summary of changes to the <a data-quicklook-from="parent-chain">parent chain</a>. This approach leverages offchain computation to boost throughput and reduce gas costs while anchoring the process to Ethereum's robust security model. To safeguard against incorrect or malicious offchain execution, Arbitrum employs a <a data-quicklook-from="challenge">challenge</a> mechanism known as fraud proofs. If a dispute arises, the STF can be recomputed step-by-step onchain, enabling the network to verify the validity of the offchain computations and ensure that the errors or fraudulent behavior are detected and rectified.
+
+Beyond these foundational aspects, the STF on the <a data-quicklook-from="arbitrum-nitro">Arbitrum Nitro</a> Stack closely mirrors Ethereum's STS in its overall structure but incorporates several key modifications to accommodate the unique requirements of the <a data-quicklook-from="arbitrum-chain">Arbitrum chain</a>. Essentially, the function receives a sequence of ordered transactions as its input, applies a defined set of rules to these inputs, and produces a new final state that reflects all the updates from the <a data-quicklook-from="batch">batch</a> of transactions.
```

Comment:
"the STF stack closely mirrors Ethereum's STS"
Are we referring to Ethereum's state trie, or is it a typo?

Reply:
Just got to this point. This is a Typo, should be STF